wpunit.suite.yml

```diff
@@ -0,0 +1,24 @@
+# Codeception Test Suite Configuration
+
+# Suite for unit or integration tests that require WordPress functions and classes.
+
+# The wpRootFolder needs to be set to your wordpress directory.
+
+actor: WpunitTester
+modules:
+    enabled:
+        - WPLoader
+        - \Helper\Wpunit
+    config:
+        WPLoader:
+            wpRootFolder: "./public/"
+            dbName: "wpTests"
+            dbHost: "localhost"
+            dbUser: "wpTests"
+            dbPassword: "wpTests"
+            tablePrefix: "wp_"
+            domain: "localhost:8000"
```

Comment:
Not sure about the single parent config. The ports on both those files should be the same. 

Are you on the dev or master branch? It shows older DB name  'wpTest'.